src/averageAges.js

```diff
@@ -36,10 +41,21 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
-}
+  const mom = people.map(person => person.mother).filter(val => val !== null);
```

Comment:
цікаве рішення для пошуку всі мам

Reply:
не сразу понятно это один объект или массив
```suggestion
  const mothers = people.map(person => person.mother).filter(x => x !== null);
```